h2o-core/src/main/java/hex/Model.java

```diff
@@ -879,7 +879,7 @@ public void printTwoDimTables(StringBuilder sb, Object o) {
             f.setAccessible(true);
             if (t != null) sb.append(t.toString(1,false /*don't print the full table if too long*/));
           } catch (IllegalAccessException e) {
-            e.printStackTrace();
+            throw new RuntimeException(e);
```

Comment:
I don't think an issue with "printer"/toString method needs halt the algorithm - it is better to log an error and add a note about the error to the string output as well